README.md

```diff
@@ -9,10 +9,7 @@ Role has been tested with NiFi versions 1.1.x, 1.2.0, 1.3.0 and 1.4.0.
 Requirements
 ------------
 
-Prior to executing this role, the NiFi distribution must be accssible on the target system at
-```{{ nifi_base_dir }}/nifi-{{ nifi_version }}/```
-  - if RPM, the RPM must be installed
-  - if tar.gz, it must be unarchived
+  - `molecule[docker]` v3 or compatible
```

Comment:
Ran into an issue during testing - our molecule version here is different from playbooks' version - we should probably update playbooks to molecule v3.